### PR TITLE
Add physiofit manager1.0.1

### PR DESCRIPTION
**Pull request by llegregam**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by llegregam** [comment]
@bgruening @yguitton @lecorguille Please review :)

File: tools/physiofit_manager/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: physiofit_manager
+owner: workflow4metabolomics
+description: Handling of physiofit input files
+categories: [Metabolomics]
+remote_repository_url: https://github.com/MetaboHUB-MetaToul-FluxoMet/PhysioFit_Data_Manager
```
Comment (bgruening) [inline tools/physiofit_manager/.shed.yml:5]: homepage_url is the thing you want to use ... remote_repo_url is this URL to this Galaxy tool

**Comment by bgruening** [comment]
I added a commit, important in there is the single-quotes of the input and output file. Everything that is a path or a free text needs to be quoted for security reasons.

**Comment by Lain-inrae** [comment]
The comments I made in https://github.com/workflow4metabolomics/tools-metabolomics/pull/233 are still applicable here

**Comment by llegregam** [comment]
@bgruening Is there something I've missed for the merge to be accepted?

**Comment by yguitton** [comment]
@lecorguille I'm not abled to review do you know why?

**Pull request merged**